packages/common/CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-v0.4.0
+## 2.0.0
+
+### Major Changes
+
+- Added version locking to adaptors that uses `openfn/language-common`
```

Comment:
This isn't a change to common though. Any adaptor who had its dependency changed should have this.

Remember that the changeset is stuff that users need to know. It's not marketing, so we don't need to justify the work. We just need to inform users what's different in this release.